packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts

```diff
@@ -286,7 +286,7 @@ export const messages = {
   empty_components: '利用できるコンポーネントがありません',
   error_auth_token: '認証トークン更新中にエラーが発生しました。',
   error_no_org_found: '組織の認証情報が見つかりませんでした。',
-  error_invalid_org_alias: '別名にはアンダースコアと英数字のみを使用できます。',
+  error_invalid_org_alias: '別名にはアンダースコア、スペースと英数字のみを使用できます。',
```

Comment:
@ntotten how can we validate this localization change?

Reply:
FYI I asked to @shunkosa the updated translation. He is the one that created these translations at first.

Reply:
Ah awesome thanks!